Lib/configparser.py

```diff
@@ -986,8 +986,11 @@ def _read(self, fp, fpname):
             first_nonspace = self.NONSPACECRE.search(line)
             cur_indent_level = first_nonspace.start() if first_nonspace else 0
             if (cursect is not None and optname and
-                cur_indent_level > indent_level):
-                cursect[optname].append(value)
+               cur_indent_level > indent_level):
```

Comment:
Bad indentation, should be aligned with the previous line:
```suggestion
                cur_indent_level > indent_level):
```